documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -1451,7 +1451,9 @@ include::{testDir}/example/ParameterizedTestDemo.java[tags=ArgumentsAggregator_w
 ==== Customizing Display Names
 
 By default, the display name of a parameterized test invocation contains the invocation
-index and the `String` representation of all arguments for that specific invocation.
+index and the `String` representation of all arguments for that specific invocation,
+each of them preceded by the parameter name if it exists and can be found.
```

Comment:
Same here. Should we add a sentence that mentions the Java compiler flag?

Reply:
Hmm yes, probably.
Something like `... each of them preceded by the parameter name if it exists and is present in the bytecode (the test code must be compiled with the -parameters compiler flag to make sure of that)`? Or maybe in a note?

Reply:
Also, I didn't explicited the case of aggregators (it's kinda implied by "if it exists"), but should I?

Reply:
How about the following?

> each of them preceded by the parameter name (unless the argument is only available via an `ArgumentsAccessor`or `ArgumentAggregator`), if present in the bytecode (for Java, test code must be compiled with the `-parameters` compiler flag).

Reply:
I just have split the sentence in two since it's a bit long